Gemfile.lock

```diff
@@ -648,47 +652,48 @@ GEM
       httpclient (>= 2.4)
       multi_json (>= 1.3.6)
       rack (>= 1.1)
-    rack-protection (2.0.1)
+    rack-protection (2.0.3)
       rack
     rack-proxy (0.6.0)
       rack
     rack-test (0.6.3)
       rack (>= 1.0)
-    rails (4.2.10)
-      actionmailer (= 4.2.10)
-      actionpack (= 4.2.10)
-      actionview (= 4.2.10)
-      activejob (= 4.2.10)
-      activemodel (= 4.2.10)
-      activerecord (= 4.2.10)
-      activesupport (= 4.2.10)
+    rails (5.0.0)
```

Comment:
__[Brakeman]__

Rails 5.0.0 content_tag does not escape double quotes in attribute values (CVE-2016-6316). Upgrade to 5.0.0.1


<!-- Comment by [Sider](https://sider.review) -->